molgenis-ontology/src/main/java/org/molgenis/ontology/sorta/meta/MatchingTaskContentEntityMetaData.java

```diff
@@ -1,31 +1,34 @@
-package org.molgenis.ontology.matching;
+package org.molgenis.ontology.sorta.meta;
 
 import static org.molgenis.data.EntityMetaData.AttributeRole.ROLE_ID;
 
 import org.molgenis.MolgenisFieldTypes.FieldTypeEnum;
 import org.molgenis.data.support.DefaultAttributeMetaData;
 import org.molgenis.data.support.DefaultEntityMetaData;
+import org.springframework.stereotype.Component;
 
+@Component
 public class MatchingTaskContentEntityMetaData extends DefaultEntityMetaData
 {
 	public final static String ENTITY_NAME = "MatchingTaskContent";
```

Comment:
So this already exists in the master and needs to be migrated to an abstract entity by dropping the repo.